modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/AssetManagerWithSecurity.java

```diff
@@ -97,15 +92,11 @@ private boolean isAuthorizedByAcl(Version version, String mpId, String action) {
     return false;
   }
 
- @Override public Snapshot takeSnapshot(String owner, MediaPackage mp) {
-    if (isAuthorizedByAcl(mp, WRITE_ACTION)) {
-      final Snapshot snapshot = super.takeSnapshot(owner, mp);
-      final AccessControlList acl = authSvc.getActiveAcl(mp).getA();
-      storeAclAsProperties(snapshot, acl);
-      return snapshot;
-    } else {
-      return chuck(new UnauthorizedException("Not allowed to take snapshot of media package " + mp.getIdentifier().toString()));
-    }
+  @Override public Snapshot takeSnapshot(String owner, MediaPackage mp) {
+    final AccessControlList acl = authSvc.getActiveAcl(mp).getA();
```

Comment:
Previously the permissions were checked here, now they're not.  Considering this is the layer where security is supposed to be enforced, why this change?